.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+<!--
+    Thanks for your contribution to the Open 3D Engine documentation! Before creating your pull
+    request, we ask you to sign off on our submission checklist to ensure that your PR comes through
+    quickly. Our reviewers' role is to make sure that your submission follows these best practices.
+
+    By submitting your pull request with DCO singoff, you agree to the Code of Conduct and
+    Open 3D Engine documentation and website licensing terms.
+-->
+
+## Change summary
+
+<!-- Provide a short description of your changes. -->
+
```

Comment:
We may be able to do this with the new GitHub Issue Forms: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

For now, I think that we should have manual entry because I don't want to over-automate just to get basic things we need in.